x-pack/test/functional/apps/dashboard/group1/reporting/screenshots.ts

```diff
@@ -227,5 +261,57 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await kibanaServer.uiSettings.replace({});
       });
     });
+
+    describe('Sample data from Kibana 7.6', () => {
```

Comment:
The test adds sample data that was saved and imported in 7.6, as [according to the original bug](https://github.com/elastic/kibana/issues/131111), the types of visualizations here tend to highlight the race condition more than Lens visualizations.